src/frontend/e2e/utils/schemas/sampleResponseDefaults.ts

```diff
@@ -0,0 +1,11 @@
+export type SampleResponseDefaults = {
+  collection_date?: string;
```

Comment:
why are all of these optional?

Reply:
This is just helper type for default values for generating response data dynamically. If I want a sample collected three days ago, then I will supply set collection_date, otherwise a random will be generated.